formatTeams.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import csv
+import random 
+
+if len(sys.argv) != 4: 
+	print("Invalid amount of arguments: Should be maxChar maxInt")
+	sys.exit()
+
+maxChar = ord(sys.argv[2])
+maxInt = int(sys.argv[3])
+
+unformatted = open(sys.argv[1], "r", encoding="utf8")
+formatted = open("sorted.csv", "w", encoding="utf8")
```

Comment:
When writing python scripts, the preferred way is to use the [argparse library](https://docs.python.org/3/library/argparse.html) which handles the type conversions as well as type checks for you (i.e. if you want an int and the user inputs a string, it'll raise the error for you without having to even deal with it!). Could you refactor the script to use argparse instead?